examples/counter.rs

```diff
@@ -17,23 +17,21 @@ use std::str::FromStr;
 use std::sync::Arc;
 
 /// Your private key file path.
-// const PRIV_KEY_PATH: &str = "PRIV_KEY_PATH";
+const PRIV_KEY_PATH: &str = "PRIV_KEY_PATH";
 
 /// Stylus RPC endpoint url.
-const RPC_URL: &str = "";
+const RPC_URL: &str = "RPC_URL";
 
 /// Deployed pragram address.
-const STYLUS_PROGRAM_ADDRESS: &str = "";
+const STYLUS_PROGRAM_ADDRESS: &str = "STYLUS_PROGRAM_ADDRESS";
 
 #[tokio::main]
 async fn main() -> eyre::Result<()> {
```

Comment:
This doesn't completely fix it.
You also need to use the dotenv crate https://crates.io/crates/dotenv to actually read the .env file with a call at the top of main to:
dotenv().ok()